homeassistant/components/nuki/lock.py

```diff
@@ -119,17 +121,13 @@ def update(self):
             except RequestException:
                 _LOGGER.warning("Network issues detect with %s", self.name)
                 self._available = False
-                return
+                continue
```

Comment:
`continue` is not necessary, you're already at the end of the loop ? 

Also, is the lock going to be updated twice ? why?

Reply:
The first update is a soft update. If that fails, we make an hard update. That is the reason for the continue

Reply:
Like all Bluetooth Communication, which is very bugy at all, is need to make a widerange search/repair what need a lot of battery drain. So we try first a update in low mode what mostly should work if the bridge is not too much away.

Reply:
But then you should break inside the `try` ?

Reply:
No because to connect over IP to the BT bridge did not means that the bridge is connected to the door. The issue is that the bridge can also return 503 and make the same exception as if you can't connect over IP.

Reply:
Okay, but your `continue` is still not doing anything here. We're at the end of the loop so are about to get to the next iteration. Your `continue` will skip to the next iteration, which is the same behavior?

Reply:
Oh nevermind, I didn't notice the break after that. ok.